src/tm_data_types/datum/waveforms/digital_waveform.py

```diff
@@ -57,19 +58,19 @@ def __setattr__(self, key: str, value: Any) -> None:
             key: The attribute name to set as a string.
             value: The value that the attribute is set to.
         """
-        if key in "y_axis_values" "y_axis_spacing" "y_axis_offset":
+        if key in "y_axis_valuesy_axis_spacingy_axis_offset":
```

Comment:
Usually I leave spaces between the searched items just to make it more readable

Reply:
Was it supposed to be implicitly concatenated strings or a list?

Reply:
A list